packages/service/src/utils/misc.ts

```diff
@@ -17,10 +18,25 @@ export const updateTsConfig = () => {
 
     fs.writeFileSync(
       tsconfigPath,
-      prettier.format(JSON.stringify(tsconfig), {
+      format(JSON.stringify(tsconfig), {
         ...JSON.parse(fs.readFileSync(runtimeRoot('.prettierrc')).toString()),
         parser: 'json',
       }),
     );
   }
 };
+
+export const webpackCompile = async (configPath: string) => {
```

Comment:
There is one problem here though, mode is not being set, so for production is fine, because if there is no mode specified webpack assumes it's production. So need to find a way to set mode. One way I could think of is creating a clientProd, severProd, etc. files and overriding the mode to production as we would set default to development.

Reply:
We can pass the mode as 2nd argument and set it on the config object

Reply:
`webpackCompile` function is taking config path and not config object, because `run` function of `parallel-webpack` expect path of config object